simplemonitor/version.py

```diff
@@ -1,4 +1,4 @@
-from importlib_metadata import version  # compat lib for < 3.8
+from importlib.metadata import version
```

Comment:
I _think_ switching to `.` from `_` is what that `3.8` comment was about? Not sure what should really be in this file. Seemed like this line was needed, but not sure if line 3 should be removed or left there commented out.

Reply:
I think that maybe goes with my earlier comment about trying a bunch of things to see what stuck - possibly in combination with supporting older Pythons at the time.